### ISSUE TITLE
Back to top link needs discernible name

### ISSUE THREAD

**Issue by thanyabegum** [pulibrary/dpul#1215]
The back to top link fails Lighthouse's accessibility check because it doesn't have text inside the anchor tag. One solution would be to add text that says "Back to Top" and then visually hide it.

<img width="513" alt="Screen Shot 2021-11-05 at 2 11 40 PM" src="https://user-images.githubusercontent.com/66143640/140559707-abb4bc74-ee2e-4de5-bd5a-6ec36095e064.png">

[Learn more about this failing Lighthouse test.](https://web.dev/link-name/?utm_source=lighthouse&utm_medium=devtools)
